### PR TITLE
Console annotated images and modified text

### PR DESCRIPTION
**Pull request by congoblue**

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -85,6 +85,12 @@ const siteConfig = {
       md.renderer.rules.button_open = custom_rules.button_open
       md.renderer.rules.button_close = custom_rules.button_close
 
+      // style diagram annotation links
+      // e.g. ((A))
+	custom_rules.keyRule(md,"annotate","\\(","\\)","annotate_open","annotate_close")
```
Comment (Farrser) [inline website/siteConfig.js]: I think that we should limit this to 1 character long @garethnunns did you remove that feature?

Comment (garethnunns) [reply website/siteConfig.js]: I didn't remove the feature - the maximum number of characters allowed in the rule is currently 40 - I could add an argument for this.

It's also worth noting that the rule wouldn't work as described in the comment... 
That rule would match `\(like this\)` **not** `((like this))`

To make it work as described this should be changed to
````js
custom_rules.keyRule(md,"annotate","((","))","annotate_open","annotate_close")
````

Would be easier to fix if it were in a branch with just one feature in...

File: website/static/custom.css [hunk 2]
```diff
@@ -68,9 +68,27 @@
     max-width: 200px;
 }
 
+ol {
+  /*border-top: 1px solid #D80029;
+  border-bottom: 1px solid #F82049;
+  font-style: italic;*/
+  background-color: rgba(216, 0, 41, 0.05);
+  border-left: 8px solid rgba(216, 0, 41, 0.2);
+  color: #555;
```
Comment (garethnunns) [inline website/static/custom.css]: I'm not sure about this - it does make some of the text hard to read, especially with blue on red and red links on red, just thinking about accessibility and usability. Also red is often associated with warnings.

File: website/static/custom.css [hunk 3]
```diff
@@ -118,6 +136,15 @@ a .softkey, a .key {
   text-transform: uppercase;
 }
 
+.annotate {
+  border-radius: 5px;
+  padding: 1px 4px;
+  border-color: rgba(216, 0, 41, 0.8);
+  color: rgba(216, 0, 41, 0.8);
```
Comment (garethnunns) [inline website/static/custom.css]: Ideally these could be solid colours (not using opacity) so when they're used in things with backgrounds, like blockquotes, they stay the same colour for consistency with the others.

File: website/static/custom.css [hunk 4]
```diff
@@ -68,9 +68,27 @@
...
+}
+
 blockquote {
   background-color: rgba(100, 188, 255, 0.3);
+  background-image: url(/docs/images/image6.png);
+  background-repeat: no-repeat;
+  background-size: 60px;
+  background-position: 5px 10px;
   border-left: 8px solid #64b0ff;
+  padding: 5px 70px;
```
Comment (garethnunns) [inline website/static/custom.css]: This is a fair bit of padding to have on mobile, if we're keeping the large **i** symbols perhaps we just have padding on the left, e.g.
````css
padding-left: 70px;
````

File: website/siteConfig.js (hunk 1 above)
Comment (congoblue) [reply website/siteConfig.js]: Hi, sorry for lumping it all together. The comment was wrong, I initially tried ((A)) but thoguht \(A\) would be better, but I forgot to change the comment.

File: website/static/custom.css (hunk 2 above)
Comment (congoblue) [reply website/static/custom.css]: Based on previous comments on the print manual, I think the block of 1-2-3 instructions should be visually tied together somehow to set it apart from the other text, but you may be right about the background. The print manuals just have a vertical line to the left - that would probably be OK. I used red as the Avolites style colour ... but it doesn't have to be.

File: website/static/custom.css (hunk 3 above)
Comment (congoblue) [reply website/static/custom.css]: Could be solid colour #D80029, this is the colour used in the images

File: website/static/custom.css (hunk 4 above)
Comment (congoblue) [reply website/static/custom.css]: Fraser suggested using an i symbol from fontawesome - presumably that could scale using a media rule. Or the i bitmap could shrink + reduce the padding. Personally I like the i to show that it's extra info/suggestion but that is just my opinion.

**Pull request merged**